sdk/include/reactos/undocshell.h

```diff
@@ -987,6 +987,11 @@ typedef struct tagEXP_VISTA_ID_LIST
 #define EXP_KNOWN_FOLDER_SIG  0xa000000b
 #define EXP_VISTA_ID_LIST_SIG 0xa000000c
 
+/***************************************************************************/
```

Comment:
For consistency purposes this should be placed in the functions section (for example at the beginning of the  "Misc. stuff").

Reply:
Ok.
■